### PR TITLE
Update to Android Studio 3.6

### PR DESCRIPTION
**Pull request by AndyScherzinger**
- [x] merge after 3.11 branch off 

see $subject

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky compile is (for unknown reasons) completely broken while it works just fine locally in Android Studio. Haven't seen any file I missed to push :/

File: scripts/analysis/findbugs-results.txt [hunk 1]
```diff
@@ -1 +1 @@
-383
+0
```
Comment (tobiasKaminsky) [inline scripts/analysis/findbugs-results.txt]: :see_no_evil: :see_no_evil: :see_no_evil: :see_no_evil:

**Comment by tobiasKaminsky** [comment]
For me our project is completely broken with new gradle as it cannot build anything due to missing lombok…

**Comment by AndyScherzinger** [comment]
Hmm, I just updated an existing checkout being compiler before, so maybe that is the issue. 😔

**Comment by tobiasKaminsky** [comment]
There is a change in gradle-plugin which seems to first compile Kotlin and then Java. 
But as Lombok generates its boilerplate (getter, setter) only during Java compilation all these functions are missing during Kotlin compile.

There is a "delombok" which removed Lombok within 5min, but then we have lots of boilerplate code again.
But moving all of this to Kotlin is also not feasible.
So I would suggest
- delombok to get gradle 3.6 running
- slowly over time migrate data classes to Kotlin
  -  do this everytime we would change/add a getter/setter

This way we prevent to add new Java/boilerplate, but do not have to do the Kotlin move at once.

**Comment by AndyScherzinger** [comment]
Sounds good to me!

**Comment by tobiasKaminsky** [comment]
@AndyScherzinger I tend to just increase lint, as we cannot handle all the fixes right now.
We should aim to remove Java over time and this will also decrease lint again…

**Comment by AndyScherzinger** [comment]
> I tend to just increase lint, as we cannot handle all the fixes right now.

I fully agree. I removed the lint-errors (so just new warnings), so we should be fine 👍

**Comment by tobiasKaminsky** [comment]
I will merge this *after* 3.11 branch off, so that we have more time to test.
Although I hope that this is not needed :-)

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky can you please bump the scores so we can get this PR merged? ❤️

**Pull request merged**